scrunch/expressions.py

```diff
@@ -305,8 +306,8 @@ def _parse(node, parent=None):
                             # For method calls, we only allow list-of-int
                             # parameters.
                             if _name == 'args' and func_type == 'method':
-                                if 'value' not in right \
-                                        or not isinstance(right['value'], list):
+                                if 'value' not in right or\
```

Comment:
This could be replaced with `not isinstance(right.get('value'), list)` instead of the `OR`ed condition. If `value` is not in the dictionary will return `None` and the condition will remain.